src/sentry/api/event_search.py

```diff
@@ -684,7 +689,29 @@ def visit_has_filter(self, node, children):
         return SearchFilter(search_key, operator, SearchValue(""))
 
     def visit_is_filter(self, node, children):
-        raise InvalidSearchQuery('"is:" queries are only supported in issue search.')
+        negation, _, _, _, search_value = children
```

Comment:
This comes from the issue search subclass. If no translators are configured then is search will be errored out as `unsupported`.